app/models/user.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+from typing import Optional
+from uuid import uuid4, UUID
+
+from sqlalchemy import Column, DateTime
+from sqlalchemy.dialects.postgresql import UUID as UUIDColumn
+from sqlalchemy.orm import relationship
+from pydantic import BaseModel
+
+from database import base, session
+
+
+class User(base):
+    __tablename__ = "user"
+
+    uuid = Column(UUIDColumn(as_uuid=True), primary_key=True, index=True, default=uuid4)
+    created_at = Column(DateTime, default=datetime.utcnow)
+
+    devices = relationship("Device", back_populates="user", passive_deletes=True)
```

Comment:
How does this identify the user? I would expect something like user_uuid?

Reply:
Thats stupid!